refund/refund.js

```diff
@@ -0,0 +1,249 @@
+        }
+        this.bnbChainDeputy = bnbChainDeputy
+        this.limit = limit;
+        this.offsetIncoming = offsetIncoming;
+        this.offsetOutgoing = offsetOutgoing;
+    }
+
+  /**
+   * Initialize the Kava client
+   * @param {String} lcdURL api endpoint for Kava's rest-server
+   * @param {String} mnemonic Kava address mnemonic
+   * @return {Promise}
+   */
+    async initKavaClient(lcdURL, mnemonic) {
+        if (!lcdURL) {
+            throw new Error("Kava's chain's rest-server url is required");
+        }
+        if (!mnemonic) {
+            throw new Error("Kava address mnemonic is required");
+        }
+
+        // Initiate and set Kava client
+        this.kavaClient = new Kava.KavaClient(lcdURL);
+        this.kavaClient.setWallet(mnemonic);
+        try {
+            await this.kavaClient.initChain();
+        } catch (e) {
+            console.log("Error: cannot connect to Kava's lcd server")
+            return
+        }
+        return this;
+    }
+
+  /**
+   * Initialize the Binance Chain client
+   * @param {String} lcdURL api endpoint for Binance Chain's rest-server
+   * @param {String} mnemonic Binance Chain address mnemonic
+   * @param {String} network "testnet" or "mainnet"
+   * @return {Promise}
+   */
+  async initBnbChainClient(lcdURL, mnemonic, network = "testnet") {
+    if (!lcdURL) {
+        throw new Error("Binance Chain's rest-server url is required");
+    }
+    if (!mnemonic) {
+        throw new Error("Binance Chain address mnemonic is required");
+    }
+
+    // Start Binance Chain client
+    this.bnbClient = await new BnbChain(lcdURL);
+    this.bnbClient.chooseNetwork(network);
+    const privateKey = bnbCrypto.getPrivateKeyFromMnemonic(mnemonic);
+    this.bnbClient.setPrivateKey(privateKey);
+    try {
+        await this.bnbClient.initChain();
+    } catch (e) {
+        console.log("Error: cannot connect to Binance Chain's lcd server")
+        return
+    }
+
+    // Load our Binance Chain address (required for refunds)
+    const bnbAddrPrefix = network == "mainnet" ? "bnb" : "tbnb"
+    this.bnbChainAddress = bnbCrypto.getAddressFromPrivateKey(privateKey, bnbAddrPrefix);
+
+    return this;
+}
+
+    /**
+     * Manages swap refunds
+     */
+    async refundSwaps() {
+        await this.refundKavaSwaps()
+        await this.refundBinanceChainSwaps()        
+    }
+
+     /**
+     * Refund any expired swaps on Kava
+     */
+    async refundKavaSwaps() {
+        const swapIDs = await this.getRefundableKavaSwaps();
+        console.log(`Kava refundable swap count: ${swapIDs.length}`)
+
+        // Fetch account data so we can manually manage sequence when posting
+        let accountData
+        try {
+            accountData = await Kava.tx.loadMetaData(this.kavaClient.wallet.address, this.kavaClient.baseURI)
+        } catch(e) {
+            console.log(e)
+        }
+
+        // Refund each swap
+        for(let i = 0; i < swapIDs.length; i++) {
```

Comment:
Can switch to a for of loop if you want here.
```javascript
for(const swapID of swapIDs) {
 ...
}
```

Reply:
https://github.com/Kava-Labs/kava-tools/pull/15/commits/2c9734ae69925fad2bbc85f791a46506316abb9f